dataset.go

```diff
@@ -38,6 +38,14 @@ func (m *DataSet) FieldByType(dsType DataSetType) (interface{}, error) {
 		return nil, ErrNilDataSet
 	}
 
+	switch dsType {
```

Comment:
I'm not sure I like either solution, but I think be cleaner to set name = keyword for dsType = Sentiment.